src/js/gpio.js

```diff
@@ -21,13 +21,6 @@ var assert  = require('assert');
 
 var dev_open = false;
 
-var GpioSettingType = {
-  kGpioPin: 'Pin',
-  kGpioPort: 'Port',
-  getLowLetter: function(type) {
-    return type === this.kGpioPin ? 'pin' : 'port';
-  }
-};
 
 function GpioError(code, operation, message) {
```

Comment:
Is it lead to infinite recursive? and where `errno` comes from? I think the code should be like followings.
```javascript
this.name = 'GpioError'
this.message = message;
return this;
```